generators/app/templates/README.md

```diff
@@ -2,16 +2,50 @@
 
 <%= projectDescription %>
 
+You can find the objective of this project and dependencies in the [ARCHITECTURE.md](ARCHITECTURE.md) file within this repository.
+
+> **Reminder:** [Making READMEs readable](https://pages.18f.gov/open-source-guide/making-readmes-readable/).
+
+## Folder Structure
+
+```
+.
+├── .github
+|   └── PULL_REQUEST_TEMPLATE.md
+├── bin
+├── config
+├── lib
+├── src
+|   ├── assets
+|   ├── models
+|   └── services
+└── tests
+```
+
+:sparkles: **.github**: This is our integration with github service, we highlight that we are going to provide a [Pull Request Template](https://help.github.com/articles/creating-a-pull-request-template-for-your-repository/).
+
+:information_source: **bin**: Find the wrappers to start the services we provide within this code here, `package.json` is going to have commands pointing to those files.
+
+:memo: **config**: Environment variables or `.env` file? This is the place to answer these type of questions.
+
+:point_right: **lib**: It's a design decision to write the modules we may export one day in this folder, so everything that you can re-utilize must be here.
+
+:heavy_check_mark: **src**: The app we write lives here. Explore the business logic and the algorithms to help us with them all here.
+
+:heavy_check_mark: **src/assets**: When we need resources that are not code, but the code needs them, like HTML files for emails templates, images, text files and so.
+
+:heavy_check_mark: **src/models**: Do you know when we need to have a module or a class to help us structuring our information, organizing inputs or outputs, so we welcome code responsible for this here.
+
+:heavy_check_mark: **src/services**: Our external dependencies will reside here, modules to use databases, APIs or any other stuff to get the job done.
```

Comment:
Services are those files that contain the business logic itself and may use some resources from `src/models`, `src/assets`, or even something from the `/lib` folder.